docs/sources/developers/plugins/metadata.md

```diff
@@ -7,30 +7,178 @@ aliases = ["/docs/grafana/latest/plugins/developing/plugin.json/"]
+| `cacheTimeout`  | boolean | false    | For data source plugins. If the `cache timeout` option should be shown in the query options section in the query editor.   |
+| `maxDataPoints` | boolean | false    | For data source plugins. If the `max data points` option should be shown in the query options section in the query editor. |
+| `minInterval`   | boolean | false    | For data source plugins. If the `min interval` option should be shown in the query options section in the query editor.    |
+
+## routes
+
+For data source plugins. Proxy routes used for plugin authentication and adding headers to HTTP requests made by the plugin. For more information, refer to [Authentication for data source plugins](https://grafana.com/docs/grafana/latest/developers/plugins/authentication/).
+
+### Properties
+
+| Property      | Type                 | Required | Description                                                                                             |
+|---------------|----------------------|----------|---------------------------------------------------------------------------------------------------------|
+| `headers`     | array                | false    | For data source plugins. Route headers adds HTTP headers to the proxied request.                        |
+| `method`      | string               | false    | For data source plugins. Route method matches the HTTP verb like GET or POST.                           |
+| `path`        | string               | false    | For data source plugins. The route path that is replaced by the route url field when proxying the call. |
+| `reqRole`     | string               | false    |                                                                                                         |
+| `reqSignedIn` | boolean              | false    |                                                                                                         |
+| `tokenAuth`   | [object](#tokenauth) | false    | For data source plugins. Token authentication section used with an OAuth API.                           |
+| `url`         | string               | false    | For data source plugins. Route url is where the request is proxied to.                                  |
+
+### tokenAuth
+
+For data source plugins. Token authentication section used with an OAuth API.
+
+#### Properties
+
+| Property | Type              | Required | Description                                                           |
+|----------|-------------------|----------|-----------------------------------------------------------------------|
+| `params` | [object](#params) | false    | For data source plugins. Params for the token authentication request. |
+| `url`    | string            | false    | For data source plugins. Url to fetch the authentication token.       |
+
+#### params
+
+For data source plugins. Params for the token authentication request.
```

Comment:
Write out "parameters" everywhere that it is just shortening the word rather than referring to something in the code.